app/main.py

```diff
@@ -2,0 +1,12 @@
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command, source, destination = command.split()
```

Comment:
Code Style: The variable names 'command', 'source', and 'destination' are a bit vague. Consider using more descriptive names.